web/src/app/components/app-board/app-board.component.css

```diff
@@ -0,0 +1,96 @@
+/* Animate items as they're being sorted. */
```

Comment:
them-ing is important in real time - refer https://www.freecodecamp.org/news/how-to-create-themes-for-your-angular-7-apps-using-css-variables-69251690e9c5/